grug/app/src/indexer.rs

```diff
@@ -0,0 +1,57 @@
+use {
+    grug_types::{BlockInfo, BlockOutcome, Tx, TxOutcome},
+    std::convert::Infallible,
+};
+
+/// This is a null indexer that does nothing.
+#[derive(Debug, Clone)]
+pub struct NullIndexer;
+
+impl NullIndexer {
+    pub fn new() -> NullIndexer {
+        NullIndexer {}
+    }
+}
+
+impl Default for NullIndexer {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl super::Indexer for NullIndexer {
```

Comment:
if possible for our types, import the type witout specify the path.
```suggestion
impl Indexer for NullIndexer {
```